extension/delta/src/include/function/delta_scan.h

```diff
@@ -1,15 +1,48 @@
 #pragma once
 
+#include "binder/binder.h"
+#include "connector/connector_factory.h"
+#include "connector/delta_connector.h"
+#include "connector/duckdb_result_converter.h"
+#include "connector/duckdb_type_converter.h"
+#include "function/table/scan_functions.h"
 #include "function/table/simple_table_functions.h"
 
 namespace kuzu {
 namespace delta_extension {
 
+using namespace function;
```

Comment:
Let's not use `using namespace` in the header file, since it also affects all files that include this file.